crates/cli/src/cli/js-tests-scaffold/index.js

```diff
@@ -2,16 +2,23 @@
 /// See the tryorama README [https://github.com/holochain/tryorama]
 /// for a potentially more accurate example
 
-const path = require('path')
+const path = require('path');
 
-const { Orchestrator, Config, combine, singleConductor, localOnly, tapeExecutor } = require('@holochain/tryorama')
+const {
+  Orchestrator,
+  Config,
+  combine,
+  singleConductor,
+  localOnly,
+  tapeExecutor,
+} = require('@holochain/tryorama');
 
 process.on('unhandledRejection', error => {
   // Will print "unhandledRejection err is not defined"
   console.error('got unhandledRejection:', error);
 });
 
-const dnaPath = path.join(__dirname, "../dist/<<DNA_NAME>>.dna.json")
+const dnaPath = path.join(__dirname, '../dist/app5.dna.json');
```

Comment:
whoops!  Gotta re-instate the `<<DNA_NAME>>`

Reply:
ohh my bad sorry